code/mod_table/mod_table.cpp

```diff
@@ -397,6 +398,16 @@ void parse_mod_table(const char *filename)
 			stuff_boolean(&Neb_affects_fireballs);
 		}
 
+		if (optional_string("$Shadow Cascade Distances:")) {
+			float dis[4];
+			stuff_float_list(dis, 4);
+			if ((dis[0] >= 0) && (dis[1] >= 0) && (dis[2] >= 0) && (dis[3] >= 0)) {
```

Comment:
The shadow algorithm probably cannot handle non-monotonically increasing values so this check should also check that the individual values are greater than the previous one:
```suggestion
			if ((dis[0] >= 0) && (dis[1] > dis[0]) && (dis[2] > dis[1]) && (dis[3] > dis[2])) {
```

Reply:
Very true, and thanks for the update suggestion!